File: src/main/java/top/zbeboy/isy/web/system/health/SystemHealthController.java
package top.zbeboy.isy.web.system.health;

import lombok.extern.slf4j.Slf4j;
import org.springframework.stereotype.Controller;
import org.springframework.web.bind.annotation.RequestMapping;

/**
 * Created by zhaoyin on 17-8-8.
 */
@Slf4j
@Controller
@RequestMapping("/web")
public class SystemHealthController {

    /**
     * 系统状况
     *
     * @return 系统状况页面
     */
    @RequestMapping("/menu/system/health")
    public String systemLog() {
        return "web/system/health/system_health::#page-wrapper";
    }
}

File: src/main/java/top/zbeboy/isy/web/system/health/SystemHealthController.kt
package top.zbeboy.isy.web.system.health

import org.springframework.stereotype.Controller
import org.springframework.web.bind.annotation.RequestMapping

/**
 * Created by zbeboy 2017-11-11 .
 **/
@Controller
@RequestMapping("/web")
open class SystemHealthController {

    /**
     * 系统状况
     *
     * @return 系统状况页面
     */
    @RequestMapping("/menu/system/health")
    fun systemLog(): String {
        return "web/system/health/system_health::#page-wrapper"
    }
}